tempesta_fw/str.c

```diff
@@ -59,36 +59,42 @@ tfw_str_del_chunk(TfwStr *str, int id)
 static TfwStr *
 __str_grow_tree(TfwPool *pool, TfwStr *str, unsigned int flag, int n)
 {
+	TfwStr *a = NULL;
+
+	TFW_DBG("str_grow_tree:start:f:%d;f:%d;d:%s;\n", str->flags, flag,
+		str->data);
```

Comment:
Please place empty line blow the debug statement. You also can make it look better:

* replace `str_grow_tree` by `%s` and put `__func__` as the first argument;

* the purpose of `start` is unclear;

* `...:flags=%x(%x) data=%.*s` is more readable than `...:f:%d;f:%d;d:%s;` (don't forget to limit length of the strings - our strings aren't zero terminated, so you can easily get crash here!).